src/test/java/it/gov/pagopa/arc/fakers/NoticeDTOFaker.java

```diff
@@ -0,0 +1,35 @@
+package it.gov.pagopa.arc.fakers;
+
+import it.gov.pagopa.arc.model.generated.NoticeDTO;
+
+import java.time.ZonedDateTime;
+
+public class NoticeDTOFaker {
+    public static NoticeDTO mockInstance(Integer bias, boolean isCart){
+        return mockInstanceBuilder(bias, isCart).build();
+    }
+    public static NoticeDTO.NoticeDTOBuilder mockInstanceBuilder(Integer bias, boolean isCart) {
```

Comment:
I would add a check to assert if all the fields are set.
This way you will verify that eventuallu new fields will be set...